src/components/Button/Button.js

```diff
@@ -1,10 +1,18 @@
 import React, {Component, PropTypes} from 'react';
+import classNames from 'classnames';
 
 class Button extends Component {
   static propTypes = {
+    animated: PropTypes.bool,
+    basic: PropTypes.bool,
     children: PropTypes.string,
+    color: PropTypes.string,
+    emphasis: PropTypes.string,
+    fluid: PropTypes.bool,
+    inverted: PropTypes.bool,
+    loading: PropTypes.bool,
```

Comment:
There are more variations of buttons than this, but I haven't been using them. I thought we could just add them as necessary.